.travis.yml

```diff
@@ -27,18 +28,18 @@ env:
 
 matrix:
   include:
-    - if: branch =~ /^(master|(autotests.*))$/ AND env(DEPLOY_VERSION) IS NOT present
-      script:
-        #- cp .env.cypress .env
-        - npm run build
-        - npx netlify deploy --prod --json --site $NETLIFY_SITE_ID --auth $NETLIFY_ACCESS_TOKEN -d build > test.json
-        - URL=`grep -i 'deploy_url' test.json | cut -c18-70`
-        - echo "$(cat test.json)"
-        - echo "Cypress test site deployed $URL. W3 $W3_LINK"
-        #- export CYPRESS_baseUrl=$URL // this is now fixed to goodtest.netlify.app
-        - export CYPRESS_w3Link=$W3_LINK
-        - cypress run --record false --browser chrome
-        #- npm run bundlesize:check
+    # - if: branch =~ /^(master|(autotests.*))$/ AND env(DEPLOY_VERSION) IS NOT present
```

Comment:
as i understood you decided to disable e2e tests via cypress ?
- does this was discussed with @sirpy ? 
- is this it's just temporary ?
- will we use cypress in the future (i remember we had plans to refuse using it)

Reply:
as @sirpy already mentioned in another comment ([comment](https://github.com/GoodDollar/GoodDAPP/pull/2689#issuecomment-721681290)) e2e have been disabled in master, I took the same changes and applied to rn for consistency